### DISCUSSION TITLE
Question on fixing bugs on older Keycloakify version

### DISCUSSION THREAD

**Discussion by anankul** [keycloakify/keycloakify#422] [Q&A]
We are currently using Keycloakify version 6.8.13 and I am currently working on adding the MFA authenticator pages to my custom theme. I am finding a few things that need improvement in LoginConfigTotp.tsx.
I am wondering if I submit the PRs with fixes, will you be able to release it as a hot-fix on 6.8.13? I am definitely planning to upgrade my theme to use the latest version v8 but I tried and ran into issues.
Until that gets sorted, can we release those fixes on 6.8.13 so I can release my theme to customers?
cc: @garronej

**Comment by garronej** [comment]
Hello @anankul,  

Thanks a lot for your willingness to contribute to the project.  

Yes, we'll be happy to incorporate your patch.  

Please submit a PR to [the v6 branch](https://github.com/keycloakify/keycloakify/tree/v6).  

I'll be able to release for v6.

**Reply by anankul** [reply]
Awesome, do I need to create issue first to submit a PR?

**Reply by garronej** [reply]
No, don't worry, we're not that strict!

**Reply by anankul** [reply]
@garronej  here is PR whenever you get chance...https://github.com/keycloakify/keycloakify/pull/442

**Reply by garronej** [reply]
Sorry it slipped through, it's released!

**Reply by anankul** [reply]
Thanks for quick review, I did respond to your suggestion in PR.
